### DISCUSSION TITLE
Telegram Dari?mikham vasat texture dorost konam va befrestam

### DISCUSSION THREAD

**Discussion by ghost** [MatinAfzal/3DICU#4] [Q&A]
Lotfan javab bedr

**Comment by MatinAfzal** [comment]
> Lotfan javab bedr

وقت بخیر!
-شما میتونید این مخزن را Fork کنید، تغییرات لازم را انجام بدید (تکسچر های خودتون رو اضافه کنید) و به عنوان Pull request برای شاخه ی beta ثبت کنید، اینجوری گیت هاب اسم شمارو به عنوان مشارکت کننده ی های پروژه ثبت میکنه و دیگران متوجه میشن.

-همچنین میتونید با مطالعه Readme به سرور دیسکورد ما مراجعه کنید و یا با ایمیل من در ارتباط باشید تا تکسچر ها را ارسال کنید (توجه داشته باشید که در این روش اسم شما به عنوان مشارکت کننده ثبت نخواهد شد).

*توجه داشته باشید که تکسچر ها ارسالی حتما (حتما) باید تماما ساخته ی خودتون باشه و در غیر این صورت درخواست شما merge نخواهد شد.

همواره از شما برای پیوستن به جامعه ما و تلاش برای مشارکت ممنونم ❤️
